pygluu-containerlib/pygluu/containerlib/persistence/sql.py

```
@@ -164,20 +166,22 @@ def create_table(self, table_name: str, column_mapping: dict, pk_column: str):
 
     def create_index(self, query):
         """Create index using raw query."""
         with self.engine.connect() as conn:
-            try:
-                conn.execute(query)
-            except Exception as exc:  # noqa: B902
-                self.adapter.on_create_index_error(exc)
+            with conn.begin():
+                try:
+                    conn.execute(text(query))
+                except Exception as exc:  # noqa: B902
```

Ruff (0.14.10)

Do not catch blind exception: `Exception`

(BLE001)